tests/release/pipelines/README.md

```diff
@@ -1,41 +1,47 @@
-		- KEYLOAK_URL_ENV:       Keycloak URL used for authentication against stage/prod cluster
-		- OFFLINE_TOKEN_ENV :    Toolchain API URL used for authentication against stage/prod cluster
-		```
-   -  The tests will run on two dedicated namespaces, so a user not part of them need to request access to the following namespaces:
-		```
-		- dev-release-team-tenant
-		- managed-release-team-tenant
-		```
-
-## How to Run tests:
-
-### 1. To run all e2e-tests in suite ensure your changes of the suite tests are saved.
-- navigate to `e2e-tests` directory 
- ```bash
- make build 
- ./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="pipelines"
- ```
-
-### 2. To run specific test we should define the test with specific label.
-[Ginkgo](https://onsi.github.io/ginkgo/#why-ginkgo) uses labels filtering, adding a label for a test will enable to run specific tests with this label. 
+# Release pipelines e2e tests suite
 
-For example: 
-If a test has the label `label-test` to a test in suite `pipelines` then the test should run using the following arguments:
+This suite contains e2e-tests for testing release pipelines from repository [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main).
+All tests must have the label `release-pipelines`.
 
-- navigate to `e2e-tests` directory 
- ```bash
- make build 
- ./bin/e2e-appstudio --ginkgo.junit-report=report.xml  --ginkgo.focus="pipelines" --ginkgo.label-filter="label-test"
+Below is the branches in [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main). All these branches will be tested per requirement.  
  ```
-
-
-
+   - production  : this branch will be used for RHTAP product environment
+   - stage       : this branch will be used for RHTAP stage environment
+   - development : this branch is the default branch for development
+  ```
+
+### The good path with pushing to Pyxis stage (push_to_external_registry.go)
+
+This test is designed to run the release pipeline `push-to-external-registry` and validates that the artifacts are successfully pushed to Pyxis.
+
+Checkpoints:
+  - Two build PipelineRuns are created in the dev namespace.
+  - The build PipelineRuns pass.
+  - The release PipelineRuns are successfully created in the managed namespace.
+  - The release PipelineRuns are expected to pass.
+  - The Releases pass.
+  - Validate that the SBOM for both components were uploaded to Pyxis stage.
+
+### The good path of FBC related tests (fbc_release.go)
+
+This test is designed to run the release pipeline `fbc_release` related test and validates that the artifacts are successfully pushed. There are two test cases currently in this file. One of them is the happy path of releasing FBC related fragments and the other is for testing FBC hotfix releasing.
+
+  #### prerequisites: 
+  - Export the following environment variables:
+   ```
```

Comment:
Why is this formatted as code? That doesn't seem right.

Reply:
I copied it from original file.  Can you please tell what your expectation is for it? Thanks

Reply:
If you look at how this file is rendered: https://github.com/redhat-appstudio/e2e-tests/blob/237e7cedab2eb1ebeb3b9cd4d8ef105bd9460f71/tests/release/pipelines/README.md

You can see that there are three code blocks (along with a button to copy the code in the top right corner of each), but this is no code, so all of these should just be normal bullet points.

Reply:
Got it, thanks!